rel/expr_tuple.go

```diff
@@ -142,12 +142,18 @@ func (e *TupleExpr) String() string { //nolint:dupl
 func (e *TupleExpr) Eval(ctx context.Context, local Scope) (Value, error) {
 	tuple := EmptyTuple
 	var err error
+	errs := Errors{}
```

Comment:
Don't modify the core behaviour of `ExprTuple.Eval` for the sake of the testing infrastructure. Reporting multiple errors will break normal error reporting. Implement `EvalTests` instead.